components/vr-audio.js

```diff
@@ -0,0 +1,46 @@
+/* globals define */
+(function(define){'use strict';define(function(require,exports,module){
+
+  document.registerElement(
+    'vr-audio',
+    {
+      prototype: Object.create(
+        VRObject.prototype, {
+          init: {
+            value: function() {
+              var src = this.getAttribute('src');
+              var loop = this.hasAttribute('loop');
+              var xPos = +this.getAttribute('x') || 0;
+              var yPos = +this.getAttribute('y') || 0;
+              var zPos = +this.getAttribute('z') || 0;
+
+              var listener = new THREE.AudioListener();
+              var firstCameraFound = document.querySelector('vr-camera').object3D;
+              if (!firstCameraFound) throw new Error('no camera');
+              firstCameraFound.add(listener);
+
+              var sound = new THREE.Audio(listener);
+              sound.load(src);
+              sound.position.set(xPos, yPos, zPos);
+              sound.setLoop(true);
+              sound.setRefDistance(1);
+              sound.panner.maxDistance = 1000;
+              sound.connect();
+              sound.source.start(0, 0);
+              this.scene.add(sound);

```

Comment:
you don't need to add it to the scene either manually. 
Just do ```this.object3D = sound;``` Positions, and adding/removing the object to the scene will be taken care for you. Sound inherits from object3D so it follows the same pattern as the other components: https://github.com/vr-components/vr-markup/blob/master/components/vr-cube.js#L13